src/lib/models/fb.ts

```diff
@@ -269,19 +285,23 @@ export default class Facebook {
      * Save post to the database
      * @param postData
      */
-    const savePost = (postData: GroupPost): void => {
+    const savePost = (postData: Post): void => {
       const allPublicationsList = getOldPublications(outputFileName!);
       allPublicationsList.push(postData);
-      if (save) {
-        fs.writeFileSync(
-          outputFileName!,
-          JSON.stringify(allPublicationsList, undefined, 4),
-          { encoding: 'utf8' },
-        );
-      }
+      fs.writeFileSync(
+        outputFileName!,
+        JSON.stringify(allPublicationsList, undefined, 4),
+        { encoding: 'utf8' },
+      );
```

Comment:
I'm not really sure but I think this change will save all questions even the `save` is false